components/contribution-flow/index.js

```diff
@@ -644,7 +646,7 @@ class ContributionFlow extends React.Component {
           >
             {!this.props.hideHeader && (
               <Box px={[2, 3]} mb={4}>
-                <ContributionFlowHeader collective={collective} isEmbed={isEmbed} />
+                <ContributionFlowHeader collective={collective} isEmbed={isEmbed} verb={verb} />
```

Comment:
I think it would be simpler if you declared a variable like `isCrypto` in this page and dealt with that from this level-down.

Reply:
Done. 👍🏽